### PR TITLE
chore(deps): more major bumps

### PR DESCRIPTION
**Pull request by ojeytonwilliams**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -120,19 +108,19 @@
...
+    "lint-staged": "10.5.4",
     "lodash": "4.17.21",
     "markdownlint": "0.21.0",
     "npm-run-all": "4.1.5",
     "ora": "5.4.0",
     "prettier": "2.2.1",
     "prismjs": "1.23.0",
     "shx": "0.3.3",
-    "sinon": "7.4.1",
+    "sinon": "^10.0.0",
```
Comment (raisedadead) [inline package.json:123]: Since you are pinning everything else please pin this too

**Review by raisedadead — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ojeytonwilliams) [reply package.json:123]: Haha, I saw that and then thought "No, Mrugesh said leave it to the bot".

Comment (ojeytonwilliams) [inline package.json:123]:
```suggestion
    "sinon": "10.0.0",
```

Comment (ojeytonwilliams) [reply package.json]: There we go, all pinned.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**